src/components/PopupWithForm.js

```diff
@@ -30,19 +40,13 @@ class PopupWithForm extends Popup {
 
   open() {
     super.open();
-    this._button.textContent = "Save";
+    this._popupForm.reset();
   }
 
   close() {
-    this._popupForm.reset();
     super.close();
+    this._popupForm.removeEventListener("submit", this._handleSubmit);
```

Comment:
You don't need it because you remove a wrong handler. You don't add such a listener to the form


Please, delete it. You will not have to remove submit handlers after correcting the comments in `Popup`